client/state/notices/actions.js

```diff
@@ -38,6 +41,13 @@ function createNotice( status, text, options = {} ) {
 	}
 }
 
+export function setRoute( path ) {
```

Comment:
>It is triggered in `boot/index`:

Ah, search failed me, I was looking for `setRoute(` . Thanks!

>but first we need to tackle routing in redux, which I would like to see :) But for now, it makes sense here IMO.

I don't think we need to solve the routing problem for this to be moved. Why not just create `client/state/routing/actions.js` and drop this function there?